docs/guides/src/main/operator/basic-deployment.adoc

```diff
@@ -164,7 +164,7 @@ CONDITION: RollingUpdate
 === Accessing the Keycloak Deployment
 
 The Keycloak deployment is, by default, exposed through a basic nginx ingress and it will be accessible through the provided hostname.
-If the default ingress doesn't fit your use-case you can disable it by setting `disableDefaultIngress: true`:
+If the default ingress doesn't fit your use-case you can disable it by setting `ingress` spec with `endabled` property to `false` value:
```

Comment:
```suggestion
If the default ingress doesn't fit your use-case, disable it by setting `ingress` spec with `enabled` property to `false` value:
```

Reply:
@DGuhr fixed, thank you!

Reply:
> jFTR – docs will be updated separately in https://github.com/keycloak/keycloak/issues/14924.
@vmuzikar I suppose I should remove this change from this PR completely, right?